### DISCUSSION TITLE
MRTK3 Preview 14 Release 💔

### DISCUSSION THREAD

**Discussion by Zee2** [microsoft/MixedRealityToolkit-Unity#11395] [Releases]
![plug](https://user-images.githubusercontent.com/5544935/214490486-a170ee06-e606-4ad5-967e-959f6e4fed89.png)

## `pre.14`: The "pro bono" update.

In light of recent news and events, we're aware the community has a lot of questions regarding the future of MRTK. We'll be making an announcement as soon as we can, and I want to personally assure everyone that the dream, ideals, and vision of MRTK lives on throughout the XR industry. We wouldn't be where we are today without the beautiful and inspiring collaboration between the open source community, the MRTK team, and the wider XR industry. We'll share more information as soon as we have it.

We are forever in your debt, and we thank you for helping us make the future of spatial computing more open, friendly, usable, and inspiring to all.

# Changelog for MRTK3 pre.14

### [pre.14] - 2022-1-24 - `4381709`
The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). The content is mirrored at [the changelog page of the Microsoft docs website](https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-overview/changelogs/mrtk3-pre-13-changelog), for now.

### Added
- Added an option to use scalar distance instead of final displacement for toggling the bounds mode. Thank you to our community contributor @akash14darshan! ([#11264](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11264))
- Added a validator rule to enforce linear color space ([#11353](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11353))

### Changed
- Moved the spatialization mixer from the core package into the example project ([#11370](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11370))
- Renamed speech-related subsystems ([#11348](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11348))
- Made `PokeInteractor` actually use the system-defined poke radius ([#11372](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11372))
- Marked existing data binding and theming package as experimental ([#11381](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11381))
- Improved subsystem query API ([#11333](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11333))

### Fixed
- Fixed incorrect assembly version for accessibility package ([#11368](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11368))
- Fixed missing nullcheck in `UGUIInputAdapterDraggable` ([#11355](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11355))
- Fixed broken pinch amount polyfill ([#11383](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11383))
- Fixed typo in `SolverHandler` + reverted prior change that broke hand constraints ([#11392](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11392))
- Fixed warnings and missing nullchecks ([#11382](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11382))
- Fixed duplicated using statement ([#11393](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11393))
- Fixed ray visuals allocing every frame ([#11344](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11344))

## Thank you

Please stay tuned. This won't be the final chapter. Keep building amazing things.

**Comment by alexanderlarsen** [comment]
Your team has been doing amazing work on MRTK. I hope you or the community find a way to finish MRTK3!

**Comment by otri** [comment]
I would absolutely be on board with a Patreon or foundation support option.  We need MRTK to be lead with a paid full time development crew. One that loves and lives in between-the-platforms kind of relationship.  My little startup Steampunk Digital would support it, but it's not enough.  I'm sure we could get some client friends of ours to support this too. 
Let's go!! ❤️ 🤠👍
